src/content/tutorials/drizzle-with-db/drizzle-with-neon.mdx

```diff
@@ -0,0 +1,241 @@
+
+This tutorial demonstrates how to use Drizzle ORM with [Neon Postgres](https://neon.tech/) database. 
+
+<Prerequisites>
+  - `Neon`, a serverless postgres database. If you do not have an existing Neon account, sign up [here](https://neon.tech). 
+    
+    Your Neon projects come with a ready-to-use Postgres database named `neondb`. We'll use this database in the following examples.
+  
+  - You should have installed Drizzle ORM and [Drizzle kit](https://orm.drizzle.team/kit-docs/overview) along with the [Neon serverless driver](https://neon.tech/docs/serverless/serverless-driver). You can do this by running the following command:
+  <Npm>
+    drizzle-orm 
+    -D drizzle-kit
+    @neondatabase/serverless
+  </Npm>
+  
+  - You should have installed the `dotenv` package for managing environment variables, and `tsx` to run typescript files. 
+  <Npm>
+    dotenv
+    tsx
+  </Npm>  
+</Prerequisites>
+
+## Setup Neon and Drizzle ORM
+
+<Steps>
+#### Create a new Neon project
+
+Log in to the [Neon Console](https://console.neon.tech/app/projects) and navigate to the Projects section. Select a project or click the `New Project` button to create a new one.
+
+#### Retrieve your Neon database connection string
+
+Navigate to the **Connection Details** section in the project console to find your database connection string. It should look similar to this:
+
+```bash
+postgres://username:password@ep-cool-darkness-123456.us-east-2.aws.neon.tech/neondb
+```
+
+Add the `DATABASE_URL` environment variable to your `.env` file, which you'll use to connect to our Neon database.
+
+```bash
+DATABASE_URL=NEON_DATABASE_CONNECTION_STRING
+```
+
+#### Connect Drizzle ORM to your database 
+
+Create a `db.ts` file and set up your database configuration:
+
+```typescript copy filename="src/db.ts"
+import { drizzle } from "drizzle-orm/neon-http";
+import { neon } from "@neondatabase/serverless";
+import { config } from "dotenv";
+
+config({ path: ".env" });
+
+const sql = neon(process.env.DATABASE_URL!);
+export const db = drizzle(sql);
+```
+
+#### Create tables
+
+Create a `schema.ts` file and declare your tables:
+
+```typescript copy filename="src/schema.ts"
+import { pgTable, integer, serial, text, timestamp } from "drizzle-orm/pg-core";
+
+export const authors = pgTable("authors", {
+  id: serial("id").primaryKey(),
+  name: text("name").notNull(),
+  bio: text("bio"),
+  createdAt: timestamp("created_at").notNull().defaultNow(),
+});
+
+export const books = pgTable("books", {
+  id: serial("id").primaryKey(),
+  title: text("title").notNull(),
+  authorId: integer("author_id").references(() => authors.id),
+  createdAt: timestamp("created_at").notNull().defaultNow(),
+});
+```
+
+The code defines two tables: `authors`, which will contain the list of all the authors, and `books`, which will contain the list of books written by the authors. Each book is associated with an author using the `authorId` field.
+
+#### Generate migrations
```

Comment:
Follow please the point `Setup Drizzle config file` in other tutorials. Links for Drizzle kit docs should be added